src/thebelab.js

```diff
@@ -237,10 +247,24 @@ function renderCell(element, options) {
   function restart() {
     let kernel = $cell.data("kernel");
     if (kernel) {
-      kernelPromise.then((kernel) => {
-        kernel.restart();
+      return kernelPromise.then(async (kernel) => {
+        await kernel.restart();
```

Comment:
```suggestion
        setOutputText("Restarting kernel...")
        await kernel.restart();
```

is adding an indication for a restarting kernel as simple as this?

Reply:
That will only work for the current cell I believe.